src/blob_file_iterator_test.cc

```diff
@@ -4,6 +4,7 @@
 
 #include "file/filename.h"
 #include "test_util/testharness.h"
+#include "util/random.h"
```

Comment:
why include this file?

Reply:
just to fix the build after updated rocksdb. too lazy to send a separate one liner fix..

Reply:
removed after rebase